docs/release.md

```diff
@@ -0,0 +1,78 @@
+# Release Process
+
+This document outlines the release process of the load-test tool.
+
+## Requirements
+
+This release process follows [the release process of the self-hosted version of Mattermost](https://handbook.mattermost.com/operations/research-and-development/product/release-process/feature-release).
+
+The requirements for this release process are the following:
+- The monthly performance comparisons, where we compare the Mattermost version about to be released with the latest stable one, are based on the RC-1 version of the new release.
+- We need a stable tested version of the load-test tool to run the performance comparisons.
+- The RC-1 version of Mattermost is cut at around T-14[^1].
+
+[^1]: Mattermost release process specifies all the release events (feature freeze, RC-1 branch cut, final build cut...) by using the T-X notation, where X is the number of working days before the release day.
+
+As such, we base the timeline of our own release process on the following goal:
+
+> We aim to have a new release ready on T-15.
```

Comment:
Historically this tool has seen periods of high and low activity hence why we never formalized on a fixed cadence.
I am thinking there's a chance we could end up skipping releases in the future due to nothing changing in the month. Not a huge deal, just something to be aware of, unless we want to bump version for the sake of it but not sure it's needed, unless of course we decide to start mapping versions with the product (e.g. `mattermost-load-test-ng` 9.0, 9.1 etc).

Reply:
> I am thinking there's a chance we could end up skipping releases in the future due to nothing changing in the month. 

Indeed, I'll clarify that skipping releases should be expected if no changes happened.

> unless we want to bump version for the sake of it but not sure it's needed, unless of course we decide to start mapping versions with the product

I don't think that's needed, no.